src/theme/text.rs

```diff
@@ -49,14 +50,23 @@ pub fn transparent(theme: &Theme) -> Appearance {
 }
 
 pub fn nickname(theme: &Theme, seed: Option<String>, transparent: bool) -> Appearance {
+    let dark_theme = theme.colors().is_dark_theme();
+
+    if seed.is_none() {
+        let mut color = theme.colors().text.base;
+        if transparent {
+            color = alpha(color, if dark_theme { 0.2 } else { 0.4 })
+        }
+        return Appearance { color: Some(color) };
+    }
```

Comment:
What about:

```rust
    if seed.is_none() {
        let color = match transparent {
            true => theme.colors().text.med_alpha,
            false => theme.colors().text.base,
        };

        return Appearance { color: Some(color) };
    }
```

Reply:
That's much nicer and more compact, will do that!